presto-product-tests/conf/docker/common/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+services:
+  hadoop-master-common:
+    hostname: hadoop-master
+    image: 'teradatalabs/cdh5-hive:2'
+    ports:
+      - '1080:1080'
+      - '8020:8020'
+      - '8088:8088'
+      - '9083:9083'
+      - '10000:10000'
+      - '50070:50070'
+      - '50075:50075'
+
+  hadoop-master-kerberos:
+    extends:
+      service: hadoop-master-common
+    image: 'teradatalabs/cdh5-hive-kerberized:2'    
```

Comment:
@sanjay990 isn't there any better way of tagging to make clear what's the differentiator in given version? E.g. `:jdk1.8.0_92`?